var/spack/repos/builtin/packages/jsoncpp/package.py

```diff
@@ -42,6 +43,17 @@ class Jsoncpp(CMakePackage):
     depends_on('cmake@3.1:', type='build')
     depends_on('python', type='test')
 
+    # Ref: https://github.com/open-source-parsers/jsoncpp/pull/1023
+    # Released in 1.9.2, patch does not apply cleanly across releases.
+    # May apply to more compilers in the future.
```

Comment:
Maybe these details aren't necessary?  I'd honestly prefer to just keep the original patch file from the PR, but I don't know if we can do it without line numbers?  (Happy to change if that can be done!  The patch file `https://github.com/open-source-parsers/jsoncpp/pull/1023.diff` breaks on `1.8.4` and earlier if it helps testing)